src/components/regboks.tsx

```diff
@@ -0,0 +1,133 @@
+                setLasterData(true);
+                const [_registrering] = await Promise.all([hentRegistrering(fnr)]);
+                setRegistrering(_registrering);
+            } catch (err) {
+                setHarFeil(true);
+            } finally {
+                setLasterData(false);
+            }
+        };
+
+        hentOverblikkData();
+    }, [fnr]);
+
+    if (lasterData) {
+        return (
+            <Panel border className="nokkelinfo_panel">
+                <Laster />
+            </Panel>
+        );
+    }
+
+    if (harFeil) {
+        return (
+            <Panel border className="nokkelinfo_panel">
+                <Errormelding />
+            </Panel>
+        );
+    }
+
+    const registrertAvNavn: StringOrNothing = registrering?.registrering?.manueltRegistrertAv?.enhet?.navn;
+    const registrertAvID: StringOrNothing = registrering?.registrering?.manueltRegistrertAv?.enhet?.id;
+    const datoRegistrert: StringOrNothing = registrering?.registrering?.opprettetDato;
+
+    const regDataHvorfor = registrering?.registrering?.teksterForBesvarelse.find(
+        (item) => item.sporsmalId === 'dinSituasjon'
+    );
+    const hvorforSvar: StringOrNothing = regDataHvorfor?.svar;
+
+    const regDataSisteStilling = registrering?.registrering?.teksterForBesvarelse.find(
+        (item) => item.sporsmalId === 'sisteStilling'
+    );
+    const sisteStillingSvar: StringOrNothing = regDataSisteStilling?.svar;
+
+    const regDataUtdanning = registrering?.registrering?.teksterForBesvarelse.find(
+        (item) => item.sporsmalId === 'utdanning'
+    );
+    const UtdanningSvar: StringOrNothing = regDataUtdanning?.svar;
+
+    const regDataUtdanningGodkjent = registrering?.registrering?.teksterForBesvarelse.find(
+        (item) => item.sporsmalId === 'utdanningGodkjent'
+    );
+    const UtdanningGodkjentSvar: StringOrNothing = regDataUtdanningGodkjent?.svar;
+
+    const regDataUtdanningBestatt = registrering?.registrering?.teksterForBesvarelse.find(
+        (item) => item.sporsmalId === 'utdanningBestatt'
+    );
+    const UtdanningBestattSvar: StringOrNothing = regDataUtdanningBestatt?.svar;
+
+    const regDataHelse = registrering?.registrering?.teksterForBesvarelse.find(
+        (item) => item.sporsmalId === 'helseHinder'
+    );
+    const HelseSvar: StringOrNothing = regDataHelse?.svar;
+
+    const regDataAnnet = registrering?.registrering?.teksterForBesvarelse.find(
+        (item) => item.sporsmalId === 'andreForhold'
+    );
+    const AnnetSvar: StringOrNothing = regDataAnnet?.svar;
+
+    const regIdNavn = registrertAvID + ', ' + registrertAvNavn;
+    const regDato = 'Registrert: ' + formaterDato(datoRegistrert);
+    const regAv = 'Registrert av: ' + regIdNavn;
+
+    const regValues = [`${regDato}`, `${regAv}`];
+
+    return (
+        <Panel border className="info_panel">
+            <Heading spacing level="2" size="large">
+                Registering
+            </Heading>
+            <span>
```

Comment:
Den her ser ikke ut til å ha noen funksjon så er fint å fjerne den for å holde strukturen så enkel som mulig